### PR TITLE
Added console command "cycle/schema/clear".

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Command/Common/SchemaClearCommand.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Yiisoft\Yii\Cycle\Command\Common;
```
Comment (roxblnfk) [inline src/Command/Common/SchemaClearCommand.php:3]:
```suggestion
<?php

declare(strict_types=1);

namespace Yiisoft\Yii\Cycle\Command\Common;
```

File: src/Command/Common/SchemaClearCommand.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    public function __construct(CycleDependencyProxy $promise)
+    {
+        $this->promise = $promise;
+        parent::__construct();
+    }
+
+    public function configure(): void
+    {
+        $this->setDescription('Clear current schema');
```
Comment (roxblnfk) [inline src/Command/Common/SchemaClearCommand.php:25]:
```suggestion
        $this->setDescription('Clear cached schema');
```

File: src/Command/Common/SchemaClearCommand.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->promise->getSchemaManager()->clear();
+        $output->writeln('Schema cleared.');
```
Comment (roxblnfk) [inline src/Command/Common/SchemaClearCommand.php:31]:
```suggestion
        $output->writeln('Schema has been removed from cache.');
```

File: src/Command/Common/SchemaClearCommand.php (hunk 2 above)
Comment (viktorprogger) [reply src/Command/Common/SchemaClearCommand.php:25]: What about "Clear curret schema cache"?

File: src/Command/Common/SchemaClearCommand.php (hunk 3 above)
Comment (viktorprogger) [reply src/Command/Common/SchemaClearCommand.php:31]:
```suggestion
        $output->writeln('Schema has been removed from cache.');
```
You don't need an article here.

File: src/Command/Common/SchemaClearCommand.php (hunk 2 above)
Comment (roxblnfk) [reply src/Command/Common/SchemaClearCommand.php:25]: > What about "Clear current schema cache"?

Also not bad

**Pull request merged**